compatibility_server/run-in-docker.sh

```diff
@@ -18,5 +18,5 @@
 
 set -e
 
-docker build -t compatibility-image .
-docker run -p 8888:8888 compatibility-image
+docker build -t compatibility-image . --export_metrics
```

Comment:
What do the flags do here?

Reply:
oh uhhh this might have been me fat fingering a double line delete.

now that I'm re-reading this line, though, I must have thought that I needed to pass the export_metrics flag when running the compatibility image in order to write error metrics to stackdriver - was this completely off the mark? I realize now that that might have been a mistake, but I'm also wondering how I can pass the flag along otherwise?

Reply:
You need to pass --export_metrics to compatibility_checker_server:app but you are already doing that in Dockerfile. AFAIK, having the flag here isn't doing anything.

You could parameterize the Dockerfile (https://docs.docker.com/engine/reference/builder/#arg) but you aren't doing that here.

Reply:
I removed the flags